lesson_3/main.cpp

```diff
@@ -0,0 +1,88 @@
+#include <iostream>
+#include <numbers>
+
+void part_1();
+void part_2();
+void part_3();
+void part_4();
+void part_5();
+
+int main() {
+    // Homework part 1:
+    part_1();
+
+    // Homework part 2:
+    part_2();
+
+    // Homework part 3:
+    part_3();
+
+    // Homework part 4:
+    part_4();
+
+    // Homework part 5:
+    part_5();
+
+    return 0;
+}
+
+void part_1() {
+    int seconds;
+
+    std::cout << "Enter number of seconds player have been in game: ";
+    std::cin >> seconds;
+
+    int hours = seconds / 3600;
```

Comment:
const values please instead of "magic" numbers to ease understanding and modification later
e.g.
const unsigned SecondsPerMinute = 60;
const unsigned MinutesPerHour = 60;
const unsigned SecondsPerHour = SecondsPerMinute * MinutesPerHour;
...

Reply:
thx, got it!